more_itertools/more.py

```diff
@@ -175,6 +197,55 @@ def peek(self, default=_marker):
                 return default
         return self._cache[0]
 
+    def prepend(self, *items):
+        """Stack up items to be the next ones returned from ``next()`` or
+        ``self.peek()``. The items will be returned in first-in first-out order:
+
+            >>> p = peekable([1, 2, 3])
+            >>> p.prepend(10, 11, 12)
+            >>> next(p)
+            10
+            >>> list(p)
+            [11, 12, 1, 2, 3]
+
+        If you want to prepend elements such that the last one in the argument
```

Comment:
Well, I put it in there precisely because I do think it's useful to remind people of this trick - people who might otherwise complain about `prepend()` not working in the other direction, or not even having an option to do so. It's obvious that you can reverse an iterable with `reversed()`, but not so obvious that you can reverse the order of an argument list with the same technique. I'm not strongly _opposed_ to removing this block, but it's not something I put in just on a whim either. I'm trying to anticipate how people who haven't read this discussion might respond to the implementation choices we're making.